config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -0,0 +1,6395 @@
+periodics:
+- tags:
+  - generated   # AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT!
```

Comment:
are these all node tests? if so why not move this generator machinery into kubernetes/sig-node/ and make yguo0905 an approver there ...?

Reply:
no not all of them... node + image validation + some release validation jobs... We can add some selector to generate multiple files in the future.

Reply:
ok